primo/opt_model/tests/test_result_parser.py

```diff
@@ -225,7 +225,7 @@ def get_minimal_campaign():
 
 @pytest.fixture
 def get_project(get_campaign):
-    return get_campaign.projects[1]
+    return get_campaign.projects[2]
```

Comment:
Curious why this test had to be changed?

Reply:
Previously, the projects attribute in the Campaign class used the project index (1, 2, 3, 4, etc.) as its keys. I have updated this to use the cluster number as the key instead, which helps prevent confusion during the override step when projects are added or removed. As a result of this change, the project currently being tested, which has an index of "1," now belongs to cluster "2."